allensdk/brain_observatory/behavior/behavior_session.py

```diff
@@ -497,12 +497,17 @@ def from_nwb(
         task_parameters = TaskParameters.from_nwb(nwbfile=nwbfile)
         trials = cls._trial_table_class().from_nwb(nwbfile=nwbfile)
         date_of_acquisition = DateOfAcquisition.from_nwb(nwbfile=nwbfile)
-        if skip_eye_tracking:
-            eye_tracking_rig_geometry = None
-            eye_tracking_table = None
-        else:
+
+        with warnings.catch_warnings():
+            warnings.filterwarnings(action='ignore',
```

Comment:
Why not delete the warning then?

Reply:
I think because I was trying to have a context for the warning? As in if you specifically wanted to load the EyeTracking in another context or by itself, you would see the warning. Don't know how much sense that makes but it was the idea.

Reply:
Cool, I'll merge as is then.